pkgs/applications/editors/hmm/default.nix

```
@@ -0,0 +1,40 @@
+{ stdenv
+, lib
+, makeWrapper
+, php
+, ncurses
+, coreutils
+, xclip
+}:
+
+stdenv.mkDerivation rec {
+  name = "h-m-m";
+
+  src = builtins.fetchGit {
+    url = "https://github.com/nadrad/h-m-m.git";
+    rev = "ecf74433844e5542f78f64f8a79807818743adff";
+  };
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -r h-m-m $out/bin/h-m-m
```

This is just from my understanding of the git repo, I'm not sure this will work at all
```suggestion
    cp $src/h-m-m $out/bin
```
Or maybe this might work wihout the postFixup lines
```nix
    cp $src/h-m-m $out/lib
    echo -e "#!/bin/sh\nphp $out/lib/h-m-m \"$@\"" > $out/bin
```